lib/options.json

```diff
@@ -655,13 +652,6 @@
         "exclude": true
       }
     },
-    "ServerString": {
-      "type": "string",
-      "minLength": 1,
-      "cli": {
-        "exclude": true
-      }
-    },
```

Comment:
In theory developer can use `type: "own-server-package"` which compatibility with connect, so let's add test for this case (just reexport express for tests)